### ISSUE TITLE
[Feature] Ability to place the data points apart from the actual point and connect it by a line.

### ISSUE THREAD

**Issue by rajkumar-osm** [chartjs/chartjs-plugin-datalabels#100]
Ability to draw a chart line/scatter and describe the each data point with a dashed line ending with a circle containing the description.

Sample:

![gnoqn](https://user-images.githubusercontent.com/17409789/48598301-40653a00-e988-11e8-9700-1cbc9d4d5298.png)

**Comment by simonbrunel** [comment]
@rajkumar-osm No, this feature is not implemented. Please use this project (chartjs-plugin-datalabels) to request features or submit bugs about data labels.
